react-sandbox/README.md

```
@@ -0,0 +1,20 @@
+Run (from repository root or inside react-sandbox):
+
+# install deps for sandbox
+
+cd react-sandbox
+pnpm install
+
+# start dev server
+
+pnpm dev
+
+Open http://localhost:5174
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)